src/read/rnglists.rs

```diff
@@ -491,32 +504,41 @@ impl<R: Reader> RngListIter<R> {
 
             let range = match raw_range {
                 RawRngListEntry::BaseAddress { addr } => {
-                    self.base_address = addr;
+                    self.base_address = try_tombstone!(addr);
```

Comment:
A tombstone for a base address must not be ignored. Instead, all offsets using that base address are also tombstones. That is, change `add_base_address` to handle this for base addresses.